src/drivers/sifive_fe310-g000_pll.c

```diff
@@ -297,6 +304,40 @@ static void configure_pll(__metal_io_u32 *pllcfg, __metal_io_u32 *plloutdiv,
 
     __METAL_ACCESS_ONCE(pllcfg) &= ~(PLL_BYPASS);
 
+    /* The PLL needs 100 usec to stabilize before we test PLL_LOCK. Since LFROSC
+     * on all targets with the FE310-G000 PLL runs at 32768 Hz, we need to wait
+     * at least
+     *
+     *   ceil(32768 ticks * 100 usec / 1000000 usec) = 4 ticks
```

Comment:
I like using dimensional analysis to validate conversions like this; let's be careful and use the right dimensions:

32768 ticks/sec * 1 sec / 1e6 µsec * 100 µsec = 3.2768 ticks/100µsec

Reply:
I just combined the `32768 ticks/sec * 1 sec/1e6 usec` into a single `32768 ticks / 1e6 usec`, since I usually do that shorthand in my own dimensional analysis, but I can make this more explicit.